.github/workflows/build.yml

```diff
@@ -36,12 +36,6 @@ jobs:
       - name: Install
         run: npm install
 
-      - name: Hydrate
```

Comment:
Hm, can this be removed?  Should it?  I'm not sure ... I feel it's a good idea to have it.  Actually, I think it's necessary after "npm install" to have the node_modules get copied everywhere.

Reply:
I think this change should be reverted.  Thoughts?

Reply:
I removed this because I added "`arc hydrate && `" to `npm test` and friends in package.json, so it should automatically hydrate now. Or is there another reason why this is still needed?

The [CI logs](https://github.com/covidatlas/li/pull/587/checks?check_run_id=1000814119#step:6:1901) show the test suite running okay, save for #586.

Reply:
ok, if it works, great, let's go ahead with it.  If it's a problem later we'll fix.  Cheers!